README.md

```diff
@@ -1 +1,87 @@
-# LF_test_task
+<h1> Test task service API </h1>
+
+<h3> REST API version </h3>
+The most recent version of the API is v1.
+
+<h2> Running Docker image <i>(Linux)</i></h2>
+
+In order to run this container you'll need docker [installed](https://docs.docker.com/install/)
+
+In the `project` directory run
+
+```shell
+$ sudo docker build -t my_docker_flask:latest .
+$ sudo docker run -d -p 5000:5000 my_docker_flask:latest
```

Comment:
Usually you don't put `sudo` in docs, people decide whether they need it. Docker may be run without `sudo`, if configured.